### DISCUSSION TITLE
Any active contributors available for hire to install ISSO?

### DISCUSSION THREAD

**Discussion by ScootRay** [posativ/isso#910] [Q&A] [answered]
No longer needed, please delete. Thanks!

**Comment by ix5** [comment, marked as answer]
Hey, the installation and embedding procedure should be fairly straightforward. Is there a specific step you require help with? Do you require help with embedding the script on your website? Or configuring something?

I'd say you spend more time trying to find someone and communicating your requirements than it'd take you to do it yourself. The [Installation Instructions](https://isso-comments.de/docs/reference/installation/#install-from-pypi) for installing on a VPS are designed to take no more than 1-2 minutes.
Since this is open-source/free software and mostly used by hobbyists, there are no dedicated consulting services that I'm aware of. If you're so limited in your time, maybe a paid service like [commento](https://commento.io/) might also be an option for you.

**Reply by ScootRay** [reply]
It was more along the lines of getting it to work with a custom version of NGINX used by RunCloud. It took a few days of head banging (and cussing) but I finally got it to work, yay!  I did try Commento earlier but didn't like it.  ISSO meets my requirements perfectly.

At any rate, I want to thank you for your contributions to keeping ISSO rolling. Muchero appreciated!

Ray

**Reply by ix5** [reply]
That's nice to hear, congratulations ;)
Thank you for the kind words, the credit belongs to all Isso contributors.

**Reply by 4044ever** [reply]
>installation and embedding procedure should be fairly straightforward.

I find it a bit cluttered. Like, it said _Install Isso with pip_ - so shouldn't be there a  `sudo apt install python3-pip` ? For debian it said: `sudo apt-get install python3-setuptools python3-virtualenv python3-dev` so in Pypi `python3-virtualenv` is not required?

I believe nginx can also be useful ;-)

In 2021 I spend a week to make it work with docker. But seems with Docker you can use it only on 1 site. Now I try again om a fresh Debian 12 server and get plenty of errors. So no straightforward. _Straightforward_ is for me when I can do line by line copy and paste and it works. This time trying for 2 days now and it's not working.
So if somebody rund isso on an external server I would love to get some guidance. My website is Hugo based.
